src/render/pic.rs

```diff
@@ -0,0 +1,78 @@
+//! PIC rendering support.
+//!
+//! # Example
+//!
+//! ```
+//! use qrcode::QrCode;
+//! use qrcode::render::pic;
+//!
+//! let code = QrCode::new(b"Hello").unwrap();
+//! let pic = code.render::<pic::Color>().build();
+//! println!("{}", pic);
+
+#![cfg(feature = "pic")]
+
+use std::fmt::Write;
+use std::marker::PhantomData;
+
+use crate::render::{Canvas as RenderCanvas, Pixel};
+use crate::types::Color as ModuleColor;
+
+/// A PIC color.
+#[derive(Copy, Clone, Default, PartialEq, Eq, PartialOrd, Ord, Hash)]
+pub struct Color<'a>(pub &'a str);
+
+impl<'a> Pixel for Color<'a> {
+    type Canvas = Canvas<'a>;
+    type Image = String;
+
+    fn default_color(color: ModuleColor) -> Self {
+        Color(color.select("#000000", "#ffffff"))
+    }
+}
+
+#[doc(hidden)]
+pub struct Canvas<'a> {
+    pic: String,
+    marker: PhantomData<Color<'a>>,
+}
+
+impl<'a> RenderCanvas for Canvas<'a> {
+    type Pixel = Color<'a>;
+    type Image = String;
+
+    fn new(width: u32, height: u32, dark_pixel: Color<'a>, light_pixel: Color<'a>) -> Self {
+        Canvas {
+            pic: format!(
+                concat!(
+                    r#"command ".if !mqrcode_fg .defcolor qrcode_fg rgb {fg}""#,
+                    "\n",
+                    r#"command ".if !mqrcode_bg .defcolor qrcode_bg rgb {bg}""#,
+                    "\n",
+                    "maxpswid={w};maxpsht={h};movewid=0;moveht=1;boxwid=1;boxht=1\n",
+                    r#"define P {{ box wid $3 ht $4 color "qrcode_fg" with .nw at $1,-$2 }}"#,
```

Comment:
```suggestion
                    r#"define p {{ box wid $3 ht $4 fill qrcode_fg with .nw at $1,-$2 }}"#,
```

"Variable names begin with a lower case letter"